src/Controller.ts

```diff
@@ -35,19 +34,31 @@ export const Controller = {
 		return manager;
 	},
 
-	getEnabledEngines: async (): Promise<RuleEngine[]> => {
+	getAllEngines: async (): Promise<RuleEngine[]> => {
 		const engines: RuleEngine[] = [];
-		const logger: Logger = await Logger.child('Controller');
```

Comment:
Do we still want these methods to do any logging? That seems like it could be helpful.